src/js/pyodide.ts

```diff
@@ -209,6 +210,14 @@ export async function loadPyodide(
      */
     indexURL?: string;
 
+    /**
+     * The base URL for where new packages will be downloaded.
```

Comment:
```suggestion
     * The file path where packages will be cached in `node.js`. If a package exists
     * `packageCacheDir` it is loaded from there, otherwise it is downloaded from
     * the JsDelivr CDN and then cached into `packageCacheDir`.
```